src/py42/sdk/queries/fileevents/filters/exposure_filter.py

```diff
@@ -138,4 +138,4 @@ class WindowTitle(FileEventFilterStringField):
     events only).
     """
 
-    _term = u"windowTitle"
+    _term = u"tabTitles"
```

Comment:
Can we adjust the doc str of `WindowTitle` to mention that it includes _all_ open Tab window titles.
The verbiage currently still sounds like it is only.

Something like:

```
Class that filters events based on the name of all the browser tabs or application windows that were
    open when a browser or other app event occurred (applies to ``read by browser or other app``
    events only).
```

Reply:
Changed as requested.